src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -36,14 +36,9 @@
 #include "ngsiNotify/Notifier.h"
 
 
-
-/* ****************************************************************************
-*
-* Select 'method' to send notifications - only one can be uncommented
-*/
-//#define SEND_BLOCKING
-#define SEND_IN_NEW_THREAD
-
+// FIME P11 #1669: move this to WS library. @fortizc please take care of this
+#define WS_SCHEME         "ws://"
+#define WS_SCHEME_LENGTH  5
```

Comment:
Ok. However, after skype talk, this is going to be done by @fortizc in a next PR, so NTC with regards to this PR.